threat_intelligence/yeti.py

```diff
@@ -87,7 +96,4 @@ def _yeti_request(self, url, data):
                               json=data,
                               headers=headers,
                               auth=requests.auth.HTTPBasicAuth(self.user, self.password))
-
-        r.raise_for_status()
```

Comment:
Seems that you are calling `raise_for_status()` after each `_yeti_request` call, so why not keep it inside the method ?

Reply:
A colleague also raised a question about this. I planned to refactor this code later today.